examples/demo-app/src/analytics.js

```diff
@@ -0,0 +1,53 @@
+import { ActionTypes } from "kepler.gl/actions";
+
+import { LOCATION_CHANGE } from "react-router-redux";
+
+const {
+  ADD_DATA,
```

Comment:
Maybe easier to not de-construct here and just use `ActionTypes. ADD_DATA` in the following function. So that people don't have to edit in 2 places when adding new action to be tracked

Reply:
Good call, yep!